djpaddle/models.py

```diff
@@ -201,19 +201,29 @@ def __str__(self):
         return "Subscription <{}:{}>".format(str(self.subscriber), str(self.id))
 
 
+class Checkout(models.Model):
+    # Backup model used to store checkout info from PaddleJS. Acts as
```

Comment:
@pyepye Let's use tripple-quotes here (""") to be consistent on how to comment classes